sdk/monitor/azure-monitor-query/README.md

```diff
@@ -139,7 +139,7 @@ try:
     if response.status == LogsQueryStatus.PARTIAL:
         error = response.partial_error
         data = response.partial_data
-        print(error.message)
+        print(error)
```

Comment:
just checking, if a user was using `error.message` before it will still continue to work (without the additional info) as is?

Reply:
Yep! `error.message` will still continue to work but just with less info (as it does currently).